src/crafting.cpp

```diff
@@ -70,6 +70,8 @@ void load_recipe( JsonObject &jsobj, const std::string & /* src */, bool uncraft
     // required
     std::string result = jsobj.get_string( "result" );
     std::string category = uncraft ? "CC_NONCRAFT" : jsobj.get_string( "category" );
+    // @todo Fix the recipes and remove this
```

Comment:
The line count wouldn't be a problem but if you want to tackle that along with normalization that would be a reasonable option.